src/utils/common.js

```diff
@@ -0,0 +1,12 @@
+const isEscKey = (evt) => (evt.key === 'Escape' || evt.key === 'Esc');
+const getRandomInteger = (a = 0, b = 1) => {
```

Comment:
Лучше не забывать отделять пробелами логические блоки
```suggestion
const isEscKey = (evt) => (evt.key === 'Escape' || evt.key === 'Esc');

const getRandomInteger = (a = 0, b = 1) => {
```